.gitignore

```diff
@@ -1,2 +1,4 @@
 /.idea
```

Comment:
Can we copy the full .gitignore from the framework project instead of adding bits at a time?